File: Pixly-BE/Pixly.Models/DTOs/PhotoBasic.cs
namespace Pixly.Models.DTOs
{
    public class PhotoBasic
    {
        public int PhotoId { get; set; }
        public string Title { get; set; }
        public string Url { get; set; }
        public string Slug { get; set; }
        public UserBasic? User { get; set; }
        public string? State { get; set; }
        public string Orientation { get; set; }
        public bool IsCurrentUserLiked { get; set; }
        public bool IsCurrentUserSaved { get; set; }
    }
}

File: Pixly-FE/src/app/core/models/DTOs/PhotoBasic.ts
import { User } from "./User";

export interface PhotoBasic {
  photoId: number;
  title: string;
  url: string;
  slug: string;
  user: User;
  state: string;
  orientation: string;
  isCurrentUserLiked: boolean;
  isCurrentUserSaved: boolean;
  uploadedAt: string;
}
